test/common/deployment_types.go

```diff
@@ -376,6 +478,25 @@ func TestStatefulSetsExpectedReplicas(t TestingTB, ctx *TestingContext) {
 		rhssoExpectedReplicas = 1
 		rhssoUserExpectedReplicas = 1
 	}
+	if rhmi.Spec.Type == string(integreatlyv1alpha1.InstallationTypeManagedApi) {
+		keycloakCR := &v1alpha1.Keycloak{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      quota.KeycloakName,
+				Namespace: NamespacePrefix + "user-sso",
```

Comment:
```suggestion
				Namespace: RHSSOUserProductNamespace,
```
Assuming rebase with the latest changes from feature branch

Reply:
Missed those :see_no_evil: